### PR TITLE
670

### PR DESCRIPTION
**Pull request by Anurag-Singh-Rajput**
## Issue that this pull request solves

 Closes: #670 

### Brief description of what is fixed or changed
** Part of SWOC 2.0 **
Fixed a 3-5 sec awkward gap that it took to populate Broadcast page by adding a Loader.
Approach :- Conditionally rendering Loader component until we get the data from fetch api and thus setting a variable with state to true once data is loaded. 
## Types of changes
UI
_Put an `x` in the boxes that apply_

- [x ] Bugfix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
**BEFORE**

Screenshot (192)

**AFTER**

Screenshot (193)

Please attach the screenshots of the changes made in case of change in user interface

### CODE REVIEW

**Comment by Anurag-Singh-Rajput** [comment]
Hey @Kajol-Kumari  I have made the necessary changes.  Hope it is as per the requirement

**Review by Ruchip16 — APPROVED** [review]
Looks good to me

**Comment by Ruchip16** [comment]
> Hey @Kajol-Kumari I have made the necessary changes. Hope it is as per the requirement

@Anurag-Singh-Rajput also could you please add a screenshot of the changes you made

**Comment by Anurag-Singh-Rajput** [comment]
Done @Ruchip16 . :)

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Anurag-Singh-Rajput can we please add the loaded to the `All Broadcast` page as well?

**Comment by Anurag-Singh-Rajput** [comment]
> @Anurag-Singh-Rajput can we please add the loaded to the `All Broadcast` page as well?
Hey @Kajol-Kumari , Done

File: frontend/src/pages/Broadcast/Component/AllBroadcasts/AllBroadcasts.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { DropMenu } from "../../../../components/util/DropMenu/index.js";
 import { Card } from "./Card/index.js";
 import { Edit } from "./Edit/index.js";
 import { END_POINT } from "./../../../../config/api";
+import LoaderFull from "../../../../components/util/Loader";
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/AllBroadcasts/AllBroadcasts.jsx:11]: Can we please a meaningful name maybe `Loader`

File: frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx [hunk 2]
```diff
@@ -7,14 +7,14 @@ import styles from "../../../Home/components/Motive/motive.module.scss";
 import "./custom-owl-carousel-style.scss";
 import { Modals } from "./Modal/index.js";
 import { END_POINT } from "./../../../../config/api";
-
+import LoaderFull from "../../../../components/util/Loader";
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx:10]: Can we please a meaningful name maybe `Loader`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Anurag-Singh-Rajput I have requested few minor changes, please incorporate it.

Also, I can see the changes from your previous PR, so from next time, please follow below approach while creating a PR-
switch to main branch -> pull the latest changes from upstream(main repo) -> create a new branch -> make the required changes -> make the pr

File: frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx (hunk 2 above)
Comment (Anurag-Singh-Rajput) [reply frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx]: Yes, Done.

**Comment by Anurag-Singh-Rajput** [comment]
> 
Thanks !! I was confused about this

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**